python/pyarrow/includes/libarrow_substrait.pxd

```diff
@@ -34,10 +50,26 @@ cdef extern from "arrow/engine/substrait/options.h" namespace "arrow::engine" no
         BEST_EFFORT \
             "arrow::engine::ConversionStrictness::BEST_EFFORT"
 
+    cdef cppclass CExtensionDetails \
+            "arrow::ending::ExtensionDetails"
```

Comment:
This should be "engine"?

Does it not error out if this is wrong?

Reply:
It doesn't error in CI either. Surprising.